admin/services/check-start.service/check-start.service.spec.js

```diff
@@ -227,19 +239,24 @@ describe('check-start.service', () => {
         expect(Object.keys(res[0].questions[0])).toContain('factor1')
         expect(Object.keys(res[0].questions[0])).toContain('factor2')
       })
+
       it('does generate and include check complete & check submit sas tokens when live checks are generated', async () => {
         const res = await checkStartService.createPupilCheckPayloads([1], 1)
-        expect(sasTokenService.generateSasToken).toHaveBeenCalledTimes(4)
```

Comment:
perhaps replace instead of delete this expectation with:
expect(sasTokenService.getTokens).toHaveBeenCalledTimes(1)
or make an isolated test to ensure the method is being called?

Reply:
@GeorgeChatzigiannis Upon reflection I have deleted the whole test as it is already covered by the `getTokens()` tests.  In this case we have already mocked the `getTokens()` call so this test was only confirming it had beeen mocked correctly.  I don't see that as adding much value.